apps/src/code-studio/progressRedux.js

```diff
@@ -428,6 +428,13 @@ export const levelsByLesson = ({stages, levelProgress, currentLevelId}) => (
   ))
 );
 
+/**
+ * Get level data for one lesson/stage
+ */
+export const levelByLesson = ({stage, levelProgress, currentLevelId}) => (
```

Comment:
This should be tested like the function above: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/code-studio/progressReduxTest.js#L563

Reply:
I added a TODO and comment with a placeholder test. If we end up using this function, I'll add a test.  If not, I'll remove the comment and placeholder and add a test for whatever we do use.